cypress/support/serviceHasPageType/index.js

```diff
@@ -0,0 +1,17 @@
+import path from 'ramda/src/path';
+import pipe from 'ramda/src/pipe';
+import servicesConfig from '../config/services';
+
+const serviceHasPageType = pageType => service =>
```

Comment:
This seems confusing to me -  checked the pipe docs and I'm still not sure I follow

Is it doing something different to this?

```
const serviceHasPageType = pageType => service => (
  path([service, 'pageTypes', pageType, 'path'], servicesConfig)
)
```

Reply:
pipe is the same as a compose function by the data flows through the argument functions from left to right instead from right to left. I think pipe is more readable than compose because of this